contrib/platform/src/com/sun/jna/platform/win32/Advapi32.java

```diff
@@ -851,7 +853,7 @@ boolean GetTokenInformation(HANDLE tokenHandle,
      *         the function fails, the return value is a nonzero error code
      *         defined in Winerror.h.
      */
-    int RegOpenKeyEx(HKEY hKey, String lpSubKey, int ulOptions,
+    int RegOpenKeyEx(HKEY hKey, NativeWin32String lpSubKey, int ulOptions,
```

Comment:
This isn't broken.  `String` is properly handled.

Neither are the many other `String` mappings in this class.  I won't comment on every one.